package.json

```diff
@@ -16,10 +16,11 @@
     },
     "dependencies": {
         "ethers": "^5.4.5",
+        "js-sha3": "^0.8.0",
```

Comment:
Is there a typed library we could use?

Reply:
Good question! We might want to find such a library given https://github.com/statechannels/dispute-game/issues/19